app/views/state_file/questions/nj_review/edit.html.erb

```diff
@@ -61,6 +61,22 @@
     </div>
   <% end %>
 
+    <% if !current_intake.claimed_as_eitc_qualifying_child_unfilled? || !current_intake.spouse_claimed_as_eitc_qualifying_child_unfilled? %>
+    <div id="eitc_qualifying_child" class="white-group">
+      <div class="spacing-below-5">
+        <% if !current_intake.claimed_as_eitc_qualifying_child_unfilled? %>
+          <p class="text--bold spacing-below-5"><%=t(".claimed_as_eitc_qualifying_child") %></p>
+          <p><%= current_intake.claimed_as_eitc_qualifying_child %></p>
+        <% end %>
+        <% if !current_intake.spouse_claimed_as_eitc_qualifying_child_unfilled? %>
+          <p class="text--bold spacing-below-5"><%=t(".spouse_claimed_as_eitc_qualifying_child") %></p>
+          <p><%= current_intake.spouse_claimed_as_eitc_qualifying_child  %></p>
+        <% end %>
+        <%= link_to t("general.edit"), StateFile::Questions::NjEitcQualifyingChildController.to_path_helper(return_to_review: "y"), class: "button--small" %>
```

Comment:
[pebble] this also needs to copy the accessible review item pattern rather than the one with accessibility flaws

Reply:
fixed and will add a note to followup ticket

Reply:
https://github.com/newjersey/affordability-pm/issues/153 links here